sdk/python/feast/data_source.py

```diff
@@ -712,17 +715,16 @@ class PushSource(DataSource):
     """
 
     name: str
-    schema: Dict[str, ValueType]
+    schema: List[Field]
     batch_source: DataSource
     timestamp_field: str
 
     def __init__(
         self,
         *,
         name: str,
-        schema: Dict[str, ValueType],
+        schema: List[Field],
         batch_source: DataSource,
-        event_timestamp_column="timestamp",
```

Comment:
are we deprecating this now in pushsource? I guess since it's in alpha it's ok to change the api immediately?

Reply:
yeah exactly